podcasts/UIViewController+Presenting.swift

```diff
@@ -1,6 +1,22 @@
 import UIKit
 
 extension UIViewController {
+    var topMostPresentedViewController: UIViewController? {
+        guard UIApplication.shared.applicationState == .active else {
+            return nil
+        }
+
+        if let presentedViewController {
```

Comment:
On iPad when I have an episode details popover opened and open player automatically enabled the player won't appear.

Adding the check below and applying the [fix from here](https://github.com/Automattic/pocket-casts-ios/pull/1088/files#r1332013567) resolves this for me. 

```suggestion
        if let presentedViewController, !presentedViewController.isBeingDismissed {
```

Reply:
Done in cd5a9fd6731cffb3708c1667f2d6136b81c72d71